### PR TITLE
Support Korean.

### PR DESCRIPTION
**Pull request by RyuaNerin**

### CODE REVIEW

File: app/js/app.js [hunk 1]
```diff
@@ -46,7 +46,7 @@
       localStorage.removeItem('lang');
     }
     else {
-      $translateProvider.preferredLanguage('en');
+      $translateProvider.preferredLanguage('ko');
```
Comment (doxxx) [inline app/js/app.js]: The default must remain English.

File: app/js/services/locale.js [hunk 2]
```diff
@@ -8,6 +8,7 @@
       en: 'English',
       de: 'Deutsch',
       fr: 'Français',
-      cn: '简体中文'
+      cn: '简体中文',
+	  ko: '한국어'
```
Comment (doxxx) [inline app/js/services/locale.js]: Spaces for indentation, please.

Comment (RyuaNerin) [reply app/js/services/locale.js]: opps

**Comment by butterflo** [comment]
The Korean translations are accurate for all pre-4.0 recipes. RyuaNerin has been hosting and maintaining an unofficial Korean version of the optimizer [here](https://ffxiv.ryuanerin.kr/#/solver), and many users including me have been using them throughout Heavensward crafting in KR server. For post-4.0 recipes, systematic parsing from a published database would be the best way to guarantee the accuracy of the translations, but no such websites are currently available for Korean item names. Still, so far the translations match with the in-game item names. I am currently in L65-66 for all crafting classes.

**Comment by RyuaNerin** [comment]
[item.exh_en.csv](https://github.com/RyuaNerin/ffxiv_exh/blob/master/item.exh_en.csv)
[item.exh_ko.csv](https://github.com/RyuaNerin/ffxiv_exh/blob/master/item.exh_ko.csv)

**Comment by doxxx** [comment]
Something which will make updating this easier in the future is to create a Korean mapping file for https://github.com/doxxx/lodestone-recipe-db-scraper like kurax did for Chinese. Then when I run the scraper to update from the Lodestone, it will preserve the Chinese and Korean translations.

**Comment by doxxx** [comment]
Deployed to beta site.

**Pull request merged**